src/kensetsu-vaults.md

```diff
@@ -0,0 +1,106 @@
+# Kensetsu Platform
+
+## Theory
+
+### Overall description of the platform
+
+Kensetsu is a lending platform with an overcollateralised stablecoin pegged to the USD. It is a MakerDAO analogue in the SORA ecosystem.
+The native token for the Kensetsu platform is the KEN token, which is used for farming and protocol rewards.
+
+### Key Features - KUSD borrowing (available collateral, parameters). Fee Calculation Process
+
+A Vault is the primary entity which operates on the Kensetsu platform. The Vault has the following parameters;
+
+- **Collateral token** — A token that operates as collateral in borrowing operations. The Kensetsu platform operates with six key tokens: XOR, VAL, TBCD, PSWAP, ETH, and DAI.
+- **Debt in borrowed token** - This shows the amount of debt that the user borrowed. The Kensetsu platform allows borrowing of KUSD tokens pegged to USD.
+- **Available to borrow** - Indicates how much a user can borrow until their vault becomes liquidated.
```

Comment:
Primary parameter is `collateral deposit` in collateral asset, `Available to borrow` is calculated as `collateral value in peg asset * liquidation LTV`.